library.properties

```diff
@@ -1,4 +1,4 @@
-name=Google Cloud IoT Core JWT
+name=Google Cloud IoT Core JWT Gal
```

Comment:
Let's leave the library properties as-is, these are used for publishing releases to Arduino for the library manager.